storybook/webpack.config.js

```diff
@@ -1,26 +1,13 @@
-// you can use this file to add your custom webpack plugins, loaders and anything you like.
-// This is just the basic way to add addional webpack configurations.
-// For more information refer the docs: https://goo.gl/qPbSyX
+// load the default config generator.
+const genDefaultConfig = require('@kadira/storybook/dist/server/config/defaults/webpack.config.js')
 
-// IMPORTANT
-// When you add this file, we won't add the default configurations which is similar
-// to "React Create App". This only has babel loader to load JavaScript.
+module.exports = function ModifyConfig(config, env) {
+  const newConfig = genDefaultConfig(config, env)
 
-module.exports = {
```

Comment:
I tried adding the loaders to the config, but they produce errors. I don't see where could they be needed. Are they needed?